algebird-core/src/main/scala/com/twitter/algebird/AveragedValue.scala

```diff
@@ -44,7 +50,16 @@ object AveragedGroup extends Group[AveragedValue] {
 
   override def isNonZero(av: AveragedValue) = (av.count != 0L)
 
-  override def negate(av: AveragedValue) = AveragedValue(-av.count, av.value)
+  override def negate(av: AveragedValue) = -av
+
+  override def sumOption(iter: TraversableOnce[AveragedValue]): Option[AveragedValue] =
+    if (iter.isEmpty) None
+    else {
+      val (c, v) = iter.foldLeft((0L, 0.0)) {
+        case ((count, sum), AveragedValue(c, v)) => (count + c, sum + (c * v))
```

Comment:
yeah, that's probably right. Probably worth removing...I guess we have to allocate a `Tuple2` anyway even if we abstract out the code. We might as well just create the `AveragedValue` directly.

so kill this?